src/Chainweb/CutDB.hs

```diff
@@ -416,10 +417,7 @@ startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
                 Right hm -> unsafeMkCut v
                     <$> case _cutDbParamsInitialHeightLimit config of
                         Nothing -> return hm
```

Comment:
I think, that even this case could interfere with the pact rewind limit (although it's very unlikely). I assume with this PR we always disable the rewind limit during pact db initialization. Is that correct?